helm/ph-ee-test/values.yaml

```diff
@@ -216,15 +193,34 @@ ph-ee-engine:
 
   channel:
     enabled: true
-    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel-test"
-    imageTag: test
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel"
+    imageTag: v1.5.1
     SPRING_PROFILES_ACTIVE: "bb"
     hostname: "channel2.sandbox.fynarfin.io"
     stub_hostname: "channel-gsma2.sandbox.fynarfin.io"
     DFSPIDS: "rhino,gorilla"
+    redis:
+      host: "127.0.0.1"
+      port: 6379
+      password: ""
```

Comment:
Empty password wouldn't work. Please overwrite it with the right password.

@fynmanoj : How should secrets be managed in child charts?